dist/ami/build_ami.sh

```diff
@@ -73,14 +73,25 @@ if [ $LOCALRPM -eq 1 ]; then
         pkg_install git
     fi
 
-    if [ ! -f dist/ami/files/$PRODUCT.x86_64.rpm ] || [ ! -f dist/ami/files/$PRODUCT-kernel-conf.x86_64.rpm ] || [ ! -f dist/ami/files/$PRODUCT-conf.x86_64.rpm ] || [ ! -f dist/ami/files/$PRODUCT-server.x86_64.rpm ] || [ ! -f dist/ami/files/$PRODUCT-debuginfo.x86_64.rpm ]; then
+    rpm_names="$PRODUCT $PRODUCT-kernel-conf $PRODUCT-conf $PRODUCT-server $PRODUCT-debuginfo"
```

Comment:
Please use arrays for lists of stuff, they also work if something has a space in it.